week03/html/contact.html

```diff
@@ -0,0 +1,51 @@
+<link href="../css/contact.css" rel="stylesheet" /> 
+<link rel="icon" type="image/png" sizes="16x16" href="../src/billyLogo.png"/>
+
+<title>Lee Gabin</title>
+
+<body>
+    <!--상단바-->
+
+    <header id="header">
+        <div class="navbar">
+        <div class="nav1">
+        <a href="../html/home.html" class="logo">
+        <img src="../src/mainLogo.png" height="70px" />
+    </a>
+    </div>
+    <div class="nav2"></div>
+    <div class="nav3">
+    <ul>
+        <li><a href="../html/home.html">Home</a></li>
+        <li><a href="../html/about.html">About</a></li>
+        <li><a href="../html/project.html">Project</a></li>
+        <li><a href="../html/activity.html">Activity</a></li>
+        <li><a href="../html/contact.html" style="color: black;">Contact</a></li>
+    </ul>
+    </div>
+    </div>
+    </header>
+
+    <!--내용-->
+    <div class="contactMe">
+        <div class="title">Contact</div>
+        <div class="mail">
+            <a class="t1">E-MAIL</a><br>
+            <a href="mailto: gb94@ewhain.net">
+            <img href="mailto: gb94@ewhain.net" src="../src/gmailLogo.png" height="150px"></a>
+        </div>
+        
+        <div class="github">
+            <a class="t2">GITHUB</a><br>
+            <a href="https://github.com/billy0904">
+            <img src="../src/githubLogo.png" height="150px"></a>
+
+        </div>
+        <div class="tistory">
+            <a class="t3">TISTORY</a><br>
+            <a href="https://billyboo.tistory.com/">
+            <img src="../src/tistoryLogo.png" height="150px"></a>
+        </div>

```

Comment:
1. 연락처 수단 "목록"이므로 `ul`, `li` 태그를 활용하면 좋겠습니다.
2. 코드 재활용성을 높일 수 있도록, 같은 디자인이 적용되는 부분은 동일한 구조로 설계하여 css 코드를 재활용합니다.

```suggestion
            <ul>
                <li>
                    <span>E-MAIL</span>
                    <a href="mailto:gb94@ewhain.net"><img src="../src/gmailLogo.png" /></a>
                </li>
                <li>
                    <span>GITHUB</span>
                    <a href="https://github.com/billy0904"><img src="../src/githubLogo.png" /></a>
                </li>
                <li>
                    <span>TISTORY</span>
                    <a href="https://billyboo.tistory.com/"><img src="../src/tistoryLogo.png" /></a>
                </li>
            </ul>
```